phoenicis-repository/src/main/java/org/phoenicis/repository/FilesystemJsonRepositoryLocationLoader.java

```diff
@@ -45,7 +45,7 @@ public FilesystemJsonRepositoryLocationLoader(String repositoryListPath, ObjectM
             try {
                 result.add(new GitRepositoryLocation.Builder()
                         .withGitRepositoryUri(new URL("https://github.com/PhoenicisOrg/scripts").toURI())
-                        .withBranch("master").build());
+                        .withBranch("5.0-alpha").build());
```

Comment:
I'm talking about this string

Reply:
I know, but how should this be solved with a configuration? Do you want do specify only the default git branch in `.properties`? I don't think this makes a lot of sense (e.g. the repository URL is still hard coded).